test/e2e/specs/default-config/recordset/add.spec.ts

```diff
@@ -0,0 +1,115 @@
+/* eslint-disable max-len */
+import { expect, test, Page } from '@playwright/test';
+
+// locators
+import RecordLocators from '@isrd-isi-edu/chaise/test/e2e/locators/record';
+import RecordeditLocators, { RecordeditInputType } from '@isrd-isi-edu/chaise/test/e2e/locators/recordedit';
+import RecordsetLocators from '@isrd-isi-edu/chaise/test/e2e/locators/recordset';
+
+// utils
+import { getCatalogID, getEntityRow } from '@isrd-isi-edu/chaise/test/e2e/utils/catalog-utils';
+import { clickNewTabLink, manuallyTriggerFocus } from '@isrd-isi-edu/chaise/test/e2e/utils/page-utils';
+import { setInputValue } from '@isrd-isi-edu/chaise/test/e2e/utils/recordedit-utils';
+
+const testParams = {
+  schema_name: 'product-recordset-add',
+  table_name: 'accommodation',
+  num_rows: 6,
+  title: 'Best Western Plus Amedia Art Salzburg',
+  rating: '3.50',
+  summary: 'The BEST WESTERN PLUS Amedia Art Salzburg is located near the traditional old part of town, near the highway, near the train station and close to the exhibition center of Salzburg.\nBEST WESTERN PLUS Amedia Art Salzburg offers harmony of modern technique and convenient atmosphere to our national and international business guest and tourists.'
+};
+
+test('Recordset add record', async ({ page, baseURL }, testInfo) => {
+  const PAGE_URL = `/recordset/#${getCatalogID(testInfo.project.name)}/${testParams.schema_name}:${testParams.table_name}`;
+
+  await test.step('should load recordset page', async () => {
+    await page.goto(`${baseURL}${PAGE_URL}`);
+    await RecordsetLocators.waitForRecordsetPageReady(page);
+
+    await expect.soft(RecordsetLocators.getRows(page)).toHaveCount(6);
+  });
+
+  await test.step('show an inline comment instead of tooltip', async () => {
+    await expect.soft(RecordsetLocators.getPageTitleInlineComment(page)).toHaveText('Recordset inline comment');
+  });
+
+  await test.step('verify the text is truncated properly based on the default config, then not truncated after clicking "more"', async () => {
+    // default config: maxRecordsetRowHeight = 160
+    // 160 for max height, 10 for padding
+    const cellHeight = 170;
+
+    const firstRow = RecordsetLocators.getRows(page).nth(0);
+    const testCell = RecordsetLocators.getRowCells(firstRow).nth(4);
+    await expect.soft(testCell).toHaveText(/... more/);
+
+    let dimensions = await testCell.boundingBox();
+    if (!dimensions) return;
```

Comment:
Why would `dimensions` be falsy? Should this block other test cases? I think it should just be a condition for the calculation that you're doing 

```
if (dimensions) {
  // the calculations might be one pixel off
   expect.soft(Math.abs(dimensions.height - cellHeight)).toBeLessThanOrEqual(1);
}
```
So it can proceed with testing the `.click()` regardless of this.

Reply:
This was how I had it initially but thought we wanted to break from this `test.step` if it was not defined.